client/README-zh.md

```diff
@@ -0,0 +1,103 @@
+本文档介绍CovenantSQL客户端的使用方式. 客户端用来创建、查询、更新和删除SQLChain以及绑定的数据库。
+
+## 开始之前
+
+确保`$GOPATH/bin`目录在环境变量`$PATH`中，执行以下命令
+
+```bash
+$ go get github.com/CovenantSQL/CovenantSQL/client
+$ go get github.com/CovenantSQL/CovenantSQL/cmd/cql-utils
+```
+
+然后在你的go代码中import 第一个`client`包。
+
+
+## 初始化一个CovenantSQL客户端
```

Comment:
Please add space between Chinese and English words

Reply:
Formatted.